lib/engine/step/borrow_train.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Step
+    class BorrowTrain < Base
```

Comment:
Is this specific to 1856, in which case it should go into 1856 (until someone else needs it)

Reply:
Yeah, the only other game I can think of with borrowing (1860) doesn't have a **choice** in what to borrow